src/problems/cvrp/local_search/local_search.cpp

```diff
@@ -494,6 +495,20 @@ void cvrp_local_search::try_job_additions(const std::vector<index_t>& routes,
         }
       }
 
+      auto smallest = std::numeric_limits<gain_t>::max();
+      auto second_smallest = std::numeric_limits<gain_t>::max();
+      size_t smallest_idx = std::numeric_limits<gain_t>::max();
```

Comment:
Should be `std::size_t` for consistency with other occurences.

Reply:
Done